### PR TITLE
weston => 13

### PR DESCRIPTION
**Pull request by satmandu**
- `crew`: Fix `prepare_package` not making adding `CREW_DEST_HOME` to `directorylist` or `filelist`, which keeps those files from being installed properly.
- Update `weston` to 13.0, using fixed `crew`...

Builds properly:
- [x] `x86_64`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=weston crew update
```

### CODE REVIEW

**Comment by satmandu** [comment]
Any idea why ` FileUtils.install 'weston.ini', "#{CREW_DEST_HOME}/.config/weston.ini", mode: 0o644` isn't working in the install section?

**Comment by satmandu** [comment]
ok it looks like files are put in `/usr/local/tmp/crew/dest/home/chronos/user/.config` but that isn't actually packaged during the build.

This is a regression...

**Comment by satmandu** [comment]
Fixed the crew `prepare_package` issue...

File: packages/weston.rb [hunk 1]
```diff
@@ -60,40 +63,33 @@ class Weston < Package
...
+  def self.install
+    system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
     File.write 'weston.ini', <<~WESTON_INI_EOF
       [core]
       xwayland=true
 
       [terminal]
       font=Cousine
-      EOF"
     WESTON_INI_EOF
```
Comment (uberhacker) [inline packages/weston.rb:83]: The weston.ini could go in `def self.build`?

Comment (uberhacker) [reply packages/weston.rb:83]: Actually, now that I think of it, that would hinder the `meson_options` assignment.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**